backend/app/helpers/spree/admin/base_helper.rb

```diff
@@ -155,6 +155,11 @@ def spree_dom_id(record)
         dom_id(record, 'spree')
       end
 
+      def admin_layout(layout = nil)
+        @admin_layout = layout if layout
+        @admin_layout
+      end
+
```

Comment:
Kind of. It's undocumented, really. This PR introduces two explicit layout options that can be passed to this.